.github/workflows/mc-depup.yml

```
@@ -0,0 +1,25 @@
+            - name: MC-DepUp
```

actionlint (1.7.7)

workflow is sequence node but mapping node is expected

(syntax-check)